.github/workflows/ci.yml

```
@@ -66,28 +49,11 @@ jobs:
     strategy:
       matrix:
         # Nextflow versions: check pipeline minimum and current latest
-        nxf_ver: ['20.04.0', '']
+        nxf_ver: ['20.10.0', '']
```

```suggestion
        nxf_ver: ['21.04.1', '']
```

to make it match the latest version as defined above maybe?